### PR TITLE
Use dynamic import of "fs" module to enable webpack builds

### PR DESCRIPTION
**Pull request by joewagner**
@asutula or @awmuncy This fixes the setup I used to reproduce #568, but can one of you take a look at this and double check that it fixes your issue using the SDK in the Studio?

### CODE REVIEW

File: src/helpers/config.ts [hunk 1]
```diff
@@ -79,21 +78,37 @@ export async function extractChainId(conn: Config = {}): Promise<number> {
...
 
+// TODO: next major we should remove the jsonFileAliases helper and expose it
+//    in a different package since it doesn't work in the browser.
+const getFsModule = (function () {
+  let fs: any;
+  return async function () {
+    // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
+    if (fs) return fs;
+
+    fs = await import(/* webpackIgnore: true */ "fs");
```
Comment (joewagner) [inline src/helpers/config.ts:99]: In addition to not importing "fs" unless the dev is using the helper, I also have to tell webpack to ignore this to get the build to work.

File: src/helpers/config.ts [hunk 2]
```diff
@@ -1,4 +1,3 @@
-import fs from "node:fs";
```
Comment (joewagner) [inline src/helpers/config.ts:1]: Details on the difference between "node:fs" and "fs" are here: https://nodejs.org/api/modules.html#core-modules
TL;DR; the `node:` prefix ignore the require cache.  In our case we don't want to do that because the SDK consumer may want to be able to leverage the require cache.

**Review by awmuncy — APPROVED** [review]
Pulled this in using NPM link, and it is all working great. LGTM!

**Comment by joewagner** [comment]
> Pulled this in using NPM link, and it is all working great. LGTM!

Excellent!  This fix is released as 4.4.2

**Comment by asutula** [comment]
Nice, good work.
